docs/release-notes/cloud/cloud-2023-11-13-jira-slack-webhooks-repo-integrations-removal.md

```diff
@@ -0,0 +1,20 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of Jira, Slack, and Webhooks repository integrations November 13, 2023
+
+On the week of November 13th 2023 we removed the option to create, configure, and delete **Jira**, **Slack**, and **Webhooks integrations for your repositories**. These integrations were available on the repository **Settings**, tab **Integration**.
+
+Existing integrations will keep working for now, we'll remove them on a second phase. However, you're no longer able to configure or delete them on the Codacy app.
```

Comment:
We could also either:

1) Link the "deactivated at a later date" to the [Removal](https://github.com/codacy/docs/pull/1944/files#diff-f96c2c2dc04b493018f9ab789dd5a9c772179eb4fb614a262eeeb8d4045e98b9R14) section
2) Specify the deactivation date here as well when we know it.

Reply:
I applied both options (link and specify date). I believe they work well together, as we have more details in that section.

Also, I'm keeping "remove" instead of "deactivate" for consistency.

```suggestion
Your existing integrations will keep working for now and [will be removed on the week of MM DD YYYY](#removal-of-existing-integrations). However, you can no longer configure or delete them on the Codacy app.<!--TODO CY-7215 Update date-->
```